py3status/modules/process_status.py

```diff
@@ -21,14 +27,14 @@
 import os
 import subprocess
 
-ERR_NO_PROCESS = 'no process name given'
-
 
 class Py3status:
     """
     """
     # available configuration parameters
     cache_timeout = 10
+    format = '{state}'
+    format_error = 'no process name given'
```

Comment:
I don't like this.

We should maybe look at errors but I'd call them `error_...` not format. anyhow I think that will be a can of worms